spec/EnableExpressErrorHandler.spec.js

```diff
@@ -0,0 +1,69 @@
+const ParseServer = require("../src/index");
+const express = require('express');
+const rp = require('request-promise');
+
+describe('Enable express error handler', () => {
+  beforeEach((done) => {
+    reconfigureServer({
+      enableExpressErrorHandler: true,
+      schemaCacheTTL: 30000
+    }).then(() => {
+      done();
+    });
+  });
+
+  it('should call the default handler in case of error, like updating a non existing object', done => {
+    const serverUrl = "http://localhost:12667/parse"
+    const appId = "anOtherTestApp";
+    const masterKey = "anOtherTestMasterKey";
+    let server;
```

Comment:
Seems written to, but never read.

Reply:
It's used to store the server to be closed later
```
.then(() => {
                  server.close(done);
                });
```